passport/passport.d.ts

```diff
@@ -89,4 +89,10 @@ declare module 'passport' {
             value: string;
         }[];
     }
+
+    interface Framework {
+        initialize(passport: Passport, options?: Object): Function;
+        authenticate(passport: Passport, name: string, options?: Object, callback?: Function): Function;
+        authorize(passport: Passport, name: string, options?: Object, callback?: Function): Function;
```

Comment:
per [comment](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/11611#issue-180084804) `authorize` should be optional